internal/appsec/remoteconfig.go

```diff
@@ -0,0 +1,41 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2022 Datadog, Inc.
+
+//go:build appsec
+// +build appsec
+
+package appsec
+
+import (
+	"encoding/json"
+
+	"gopkg.in/DataDog/dd-trace-go.v1/internal/log"
+	"gopkg.in/DataDog/dd-trace-go.v1/internal/remoteconfig"
+
+	rc "github.com/DataDog/datadog-agent/pkg/remoteconfig/state"
+)
+
+// asmFeaturesCallback deserializes an ASM_FEATURES configuration received through remote config
+// and starts/stops appsec accordingly. Used as a callback for the ASM_FEATURES remote config product.
+func asmFeaturesCallback(u remoteconfig.ProductUpdate) {
+	if l := len(u); l > 1 {
+		log.Debug("%d configs received for ASM_FEATURES. Expected one at most, returning early", l)
+		return
+	}
+
+	for path, raw := range u {
```

Comment:
When reading your tests, I wondered if it would ever make sense to receive several files at once and iterate over them 🤔 But I guess we are all good with this "dummy" handler.

Reply:
I don't know how it could make sense. There is no way to prioritize between configs given said configs are a simple boolean flag. I agree it could make sense to have a default global config that, say, enables ASM, and a specific service's config that wants to disable it - or the other way around, but then we'd need a way to know which one to use.